rarbgapi/rarbgapi.py

```diff
@@ -127,6 +127,8 @@ def wrapper(self, *args, **kwargs):
                 resp = func(self, token=self._token, *args, **kwargs)
                 json_ = resp.json(object_hook=json_hook)
                 return json_['torrent_results']
+            except KeyError:
```

Comment:
Do you fix the problem that server returns `{"error":"No results found","error_code":20}` if nothing found?
If yes, how about raise a notfound exception in json_hook and functions should take care this by themselves. The request decorator is used to retry for network/expire error and just return an empty list if keyerror raising is too implicit.